changeset:   3141:d8bd17248789
tag:         tip
user:        "Yann E. MORIN" <yann.morin.1998@free.fr>
date:        Sun Jan 06 12:08:33 2013 +0100
files:       scripts/showTuple.sh.in
description:
scripts/showTuple: fix stdout

Since we've had the debug shell feature, fd #7 is now used to
redirect stderr, while it was previously unused.

Use fd #9 to redirect stdout.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>


diff --git a/scripts/showTuple.sh.in b/scripts/showTuple.sh.in
--- a/scripts/showTuple.sh.in
+++ b/scripts/showTuple.sh.in
@@ -7,7 +7,7 @@
 . "${CT_LIB_DIR}/paths.sh"
 
 # We'll need the stdout later, save it
-exec 7>&1
+exec 9>&1
 
 # Parse the common functions
 . "${CT_LIB_DIR}/scripts/functions"
@@ -27,4 +27,4 @@
 CT_DoBuildTargetTuple
 
 # All this for this single echo... :-( Sigh, I'll have to re-arrange things...
-echo "${CT_TARGET}" >&7
+echo "${CT_TARGET}" >&9

